app/src/main/java/com/puzzlebench/clean_marvel_kotlin/data/service/LoadLocalCharactersImpl.kt

```diff
@@ -0,0 +1,14 @@
+package com.puzzlebench.clean_marvel_kotlin.data.service
+
+import com.puzzlebench.clean_marvel_kotlin.data.mapper.CharacterMapperRealm
+import com.puzzlebench.clean_marvel_kotlin.domain.contracts.CharacterRepository
+import com.puzzlebench.clean_marvel_kotlin.domain.model.Character
+import com.puzzlebench.clean_marvel_kotlin.domain.model.CharacterRealm
+import io.realm.Realm
+
+class LoadLocalCharactersImpl(private val mapper: CharacterMapperRealm = CharacterMapperRealm()): CharacterRepository.Load {
```

Comment:
move this to another package like a repository package instead of this service package

Reply:
done!